llvm/unittests/Support/CommandLineTest.cpp

```diff
@@ -525,6 +525,58 @@ TEST(CommandLineTest, LookupFailsInWrongSubCommand) {
   EXPECT_FALSE(Errs.empty());
 }
 
+TEST(CommandLineTest, SubcommandOptions) {
```

Comment:
There are other Subcommand tests. The name `SubcommandOptions` can be changed to reflect that the test intends to check, e.g. `TopLevelOptInSubcommand`